client/js/layouts/index.js

```diff
@@ -0,0 +1 @@
+// Add any code here
```

Comment:
What is this dir for?

Reply:
For the same structure screens but different data fetching from API

Reply:
I don't think you need this dir. We can add it later if needed. API calls should happen in the container file.

Reply:
Ok, I removed it